src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/court/VideoLinkBookingService.kt

```diff
@@ -0,0 +1,313 @@
+package uk.gov.justice.digital.hmpps.whereabouts.services.court
```

Comment:
Renamed from CourtService.
Added CourtService as a dependency.
Only changes are to add lookups for court name -> court Id and court id -> court name when creating bookings and when retrieving bookings for client queries.